validator/client/attest.go

```diff
@@ -127,25 +139,42 @@ func (v *validator) SubmitAttestation(ctx context.Context, slot primitives.Slot,
 		return
 	}
 
-	aggregationBitfield := bitfield.NewBitlist(uint64(len(duty.Committee)))
-	aggregationBitfield.SetBitAt(indexInCommittee, true)
-	attestation := &ethpb.Attestation{
-		Data:            data,
-		AggregationBits: aggregationBitfield,
-		Signature:       sig,
+	// TODO: Extend to Electra
```

Comment:
What is this TODO?

Reply:
This is slasher logic which hasn't been updated to Electra yet. It is non-trivial and needs more discussion.